src/main/java/firebase/sdk/dashboard/dao/UserDao.java

```diff
@@ -0,0 +1,54 @@
+// TODO: Not entirely sure how interacting with user data will be but created a boiler plate.
+package firebase.sdk.dashboard.dao;
+
+import firebase.sdk.dashboard.data.User;
+import firebase.sdk.dashboard.data.SDK;
+import java.util.Map;
+
+/**
+ * An interface for retrieving and manipulating data about Users.
+ */
+public interface UserDao {
+
+  /** 
+   * Retrieves a User object from a data source.
+   *
+   * @param id The unique id that identifies the user.
+   * @return A User object containg data that represents the user and their preferences.
```

Comment:
We dont store the preferences anymore

Reply:
Since we removed the getUserFavorites from the dao because it was too high level, we now store the preferences

Reply:
@ankitaj224 is there an issue with storing the preferences in the User object. We could do it other ways even if we change the method.

Reply:
I am unclear what do you mean by preferences here?

Reply:
At the initial time of writing I was thinking that there may need to be some type of configuration for custom email settings and so I was thinking that this would also be included in the user object but right now the only thing other than the uid and email would be the users favorites. I will change the javadoc to reflect that